falcon/testing/test_case.py

```diff
@@ -18,36 +18,79 @@
 utilities for simulating and validating HTTP requests.
 """
 
-try:
-    import testtools as unittest
-except ImportError:  # pragma: nocover
-    import unittest
+import os
+import unittest
+import warnings
 
 import falcon
 import falcon.request
-
-# TODO hoist for backwards compat. Remove in falcon 4.
```

Comment:
Please don't remove this note!
We missed it in Falcon 4.0 it seems, so it needs to be updated to say Falcon 5.0, if you want to touch it.